antora-playbook.yml

```diff
@@ -20,7 +20,7 @@ content:
 output:
   destinations:
     - clean: true
-      path: build/site
+      path: out/docs
```

Comment:
we need to add the out dir to gitignore, its adding all the built files to the PR